src/platform/imx8m/lib/dai.c

```diff
@@ -18,6 +18,17 @@ static SHARED_DATA struct dai sai[] = {
 	.index = 3,
 	.plat_data = {
 		.base = SAI_3_BASE,
+		.fifo[SOF_IPC_STREAM_PLAYBACK] = {
+			.offset = SAI_3_BASE + REG_SAI_TDR0,
+			.depth = 16,
+			/* Handshake is SDMA hardware event */
```

Comment:
Looking in the upstream SAI Linux driver (sound/soc/fsl/fsl_sai.c) depth for i.MX8MQ is 128. I think that SAI is the same on 8MP vs 8MP.

Reply:
Depth is in number of words and is interpreted by SDMA driver. Will need to check.